Mathlib/Data/Matrix/Mul.lean

```diff
@@ -789,7 +789,21 @@ theorem one_vecMul [Fintype m] (A : Matrix m n α) : 1 ᵥ* A = ∑ i, A i := by
 
 @[deprecated (since := "2025-01-26")] alias vec_one_mul := one_vecMul
 
-variable [Fintype m] [Fintype n] [DecidableEq m]
+variable [Fintype m] [DecidableEq m]
+
+lemma ext_of_mulVec {M N : Matrix n m α} (h : ∀ i, M *ᵥ Pi.single i 1 = N *ᵥ Pi.single i 1) :
+    M = N := by
+  ext i j
+  simp_rw [mulVec_single_one] at h
+  exact congrFun (h j) i
```

Comment:
```suggestion
lemma ext_of_mulVec [Fintype n] [DecidableEq n] {M N : Matrix m n α} (h : ∀ i, M *ᵥ Pi.single i 1 = N *ᵥ Pi.single i 1) :
    M = N := by
  ext i j
  simp_rw [mulVec_single_one] at h
  exact congrFun (h j) i
```
so as not to be consistent about using `Matrix m n` and not `Matrix n m` throughout the file.